test/functional/cmdLineTests/shareClassTests/SCCMLTests/ShareClassesCMLTests-5.xml

```diff
@@ -835,6 +836,35 @@
 		<output type="failure" caseSensitive="no" regex="no">corrupt</output>
 		<output type="failure" caseSensitive="yes" regex="no">Processing dump event</output>
 	</test>
+
+	<!-- Test fix for https://github.com/eclipse-openj9/openj9/issues/19378 -->
+	<test id="Test 200a: reset cache and run program with utils jar file that is missing the typical .jar ending">
+		<exec command="cp $UTILSJAR$ $UTILSDIR$$PATHSEP$utils"/>
+		<command>$JAVA_EXE$ $currentMode$,reset -cp $UTILSDIR$$PATHSEP$utils -Xnolinenumbers $PROGRAM_HANOI$</command>
+		<output type="success" caseSensitive="yes" regex="no">Puzzle solved!</output>
+		<output type="failure" caseSensitive="no" regex="no">Unhandled Exception</output>
+		<output type="failure" caseSensitive="yes" regex="no">Exception:</output>
+		<output type="failure" caseSensitive="no" regex="no">corrupt</output>
+		<output type="failure" caseSensitive="yes" regex="no">Processing dump event</output>
+	</test>
+
+	<test id="Test 200b: run program with utils jar file that is missing the typical .jar ending">
+		<command>$JAVA_EXE$ $currentMode$ -cp $UTILSDIR$$PATHSEP$utils -Xnolinenumbers $PROGRAM_HANOI$</command>
+		<output type="success" caseSensitive="yes" regex="no">Puzzle solved!</output>
+		<output type="failure" caseSensitive="no" regex="no">Unhandled Exception</output>
+		<output type="failure" caseSensitive="yes" regex="no">Exception:</output>
+		<output type="failure" caseSensitive="no" regex="no">corrupt</output>
+		<output type="failure" caseSensitive="yes" regex="no">Processing dump event</output>
+	</test>
+
+	<test id="Test 200c: verify that classes from the utils jar file were reused">
```

Comment:
I suggest to change "verify that classes from the utils jar file were reused" to "verify that no classes are marked stale and classes from the utils jar file were reused".